src/bindings.cpp

```diff
@@ -1108,22 +1189,58 @@ extern "C"
     }
 
     /////////// JUST FOR TEST //////////
```

Comment:
Maybe we should get rid of these before a more stable release?

Reply:
Of course!! Forgot this